backend/Pipfile

```diff
@@ -34,7 +35,7 @@ qrcode = "*"
 python-dateutil = "*"
 psycopg2 = "*"
 django-simple-history = "*"
-channels = "<3"
+channels = "*"
```

Comment:
Can you just confirm we're not depending on a version pin here?

Reply:
I believe [this](https://github.com/pennlabs/penn-clubs/blob/eec217228c589d91987661811c9c0357ffa502db/backend/clubs/routing.py#L7) was the only part of our code that needed to be updated, but I'll do further checks/tests locally to make sure unpinning the version doesnt cause any issues